src/styles/card.scss

```diff
@@ -0,0 +1,119 @@
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  box-sizing: border-box;
+
+  background: #FFF;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+
+  margin: 0 auto;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  margin-top: 0;
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-accent);
+}
+
+.card__code {
+  margin-bottom: 16px;
+  margin-top: 0;
```

Comment:
I see that you often write  margin-top: 0; margin: 0;

you can create a file called reset.scss connect this file in the same as other tour scsss files and inside this file write the next reset rules:
```

body,
p,
h2 {
  margin: 0;
}
```

dont forget to delete body { margin: 0; } inside card.scss, there is need to duplicate code